CMake/External_Libsvm.cmake

```diff
@@ -0,0 +1,62 @@
+# Copyright 2012 2018 by Kitware, Inc. All Rights Reserved. Please refer to
+# KITWARE_LICENSE.TXT for licensing information, or contact General Counsel,
+# Kitware, Inc., 28 Corporate Drive, Clifton Park, NY 12065.
+
+# Locate the system installed Libsvm
+# The following variables will be set:
+#
+# LIBSVM_FOUND       - Set to true if Libsvm can be found
+# LIBSVM_INCLUDE_DIR - The path to the Libsvm header files
+# LIBSVM_LIBRARY     - The full path to the Libsvm library
+
+INCLUDE(FindPythonInterp)
+
+###
+# Private helper function to execute `python -c "<cmd>"`
+#
+# Runs a python command and populates an outvar with the result of stdout.
+# Be careful of indentation if `cmd` is multiline.
+#
+function(_pycmd outvar cmd)
+    execute_process(
+            COMMAND "${PYTHON_EXECUTABLE}" -c "${cmd}"
+            RESULT_VARIABLE _exitcode
+            OUTPUT_VARIABLE _output)
+    if(NOT ${_exitcode} EQUAL 0)
+        message(ERROR "Failed when running python code: \"\"\"
+${cmd}\"\"\"")
+        message(FATAL_ERROR "Python command failed with error code: ${_exitcode}")
+    endif()
+    # Remove supurflous newlines (artifacts of print)
+    string(STRIP "${_output}" _output)
+    set(${outvar} "${_output}" PARENT_SCOPE)
+endfunction()
```

Comment:
I moved it to the CMakeLists.txt which is being copied.

Reply:
Sorry, I wasn't clear. It should not go in the CMakeLists.txt file. There is a CMake/ directory which usually contains utility functions. For now, we have Utils.cmake, poorly named, I'll admit, but this functionality belongs in there for now.

Reply:
Yeah, I saw the util file, but since we're copying `CMakeLists.txt` now to the source code of libsvm, I was thinking that the `CMakeLists.txt` should be a standalone file, i.e., one should be able to run it independent of fletch to compile libsvm code.  So, I copied the relevant part of the function in that file.

Reply:
Ok, that make sense if this is a libSVM function. The way it was originally written, it was a fletch function. All of those commits are apparently gone which makes continuing the original review difficult.